### PR TITLE
Fix version local

### PR DESCRIPTION
**Pull request by outscale-vbr**
Fix version local

### CODE REVIEW

**Comment by outscale-mdr** [comment]
Could you remove unused lines in [.github/workflows/e2e_test.yml](https://github.com/outscale/cloud-provider-osc/blob/b620f4c8ab2950131269e27a4421703e36c52f31/.github/workflows/e2e_test.yml#L70) to avoid confusion ?

**Comment by outscale-vbr** [comment]
Yes

File: .github/workflows/e2e_test.yml [hunk 1]
```diff
@@ -67,8 +67,6 @@ jobs:
         public_cloud: "false"
         image_id: ${{ secrets.OMI_ID }}
         version: ${{ format('{0}.{1}', github.sha, github.run_attempt) }}
-        rke_version: "v1.4.12"
-        kubernetes_version: "v1.26.11-rancher2-1"
         bastion_vm_type: ${{ secrets.BASTION_VMTYPE }}
         worker_vm_type: ${{ secrets.WORKER_VMTYPE }}
         control_plane_vm_type: ${{ secrets.CP_VMTYPE }}
```
Comment (outscale-mdr) [inline .github/workflows/e2e_test.yml:72]: This is not used, right ?

**Review by outscale-mdr — CHANGES_REQUESTED** [review]

File: .github/workflows/e2e_test.yml (hunk 1 above)
Comment (outscale-vbr) [reply .github/workflows/e2e_test.yml:72]: Yes i remove too

**Comment by outscale-vbr** [comment]
@outscale-mdr ARe you ok with the latest changes ?

**Review by outscale-mdr — APPROVED** [review]

**Pull request merged**